packages/client/src/hooks/useSuccessMessage.js

```diff
@@ -0,0 +1,41 @@
+import { useModalContext } from "contexts";
+import { TransactionSuccessModal } from "modals";
+import { ASSET_TYPES, ACTION_TYPES } from "../constants/enums";
+
+export default function useSuccessMessage() {
+    const { openModal, closeModal } = useModalContext();
+
+    const showSuccessTransactionModal = (action, safeName, safeAddress) => {
+        const actionData = action.data.actionView;
+        openModal(
+            <TransactionSuccessModal
+                safeName={safeName}
+                safeAddress={safeAddress}
+                actionData={actionData}
+                actionType={getActionType(actionData.type)}
```

Comment:
since `actionType` belongs to `actionData`, do we have to pass `actionType` here?

Reply:
No need, so I moved it to the modal